hamocc/mo_hamocc4bcm.F90

```diff
@@ -0,0 +1,397 @@
+    use mo_biomod,        only: strahl
+    use mo_control_bgc,   only: ldtrunbgc,dtbgc,ldtbgc,io_stdo_bgc,dtbgc,ndtdaybgc, &
+                                do_sedspinup,sedspin_yr_s,sedspin_yr_e,sedspin_ncyc, &
+                                use_BROMO, use_CFC, use_PBGC_CK_TIMESTEP,&
+                                use_BOXATM, use_sedbypass,ocn_co2_type
+    use mo_param1_bgc,    only: iatmco2,iatmdms,nocetra,nriv,iatmbromo
+    use mo_vgrid,         only: set_vgrid
+    use mo_apply_fedep,   only: apply_fedep
+    use mo_apply_rivin,   only: apply_rivin
+    use mo_apply_ndep,    only: apply_ndep
+    use mo_apply_oafx,    only: apply_oafx
+    use mo_boxatm,        only: update_boxatm
+    use mo_inventory_bgc, only: inventory_bgc
+    use mo_sedshi,        only: sedshi
+    use mo_get_cfc,       only: get_cfc
+    use mo_powach,        only: powach
+    use mo_preftrc,       only: preftrc
+    use mo_cyano,         only: cyano
+    use mo_ocprod,        only: ocprod
+    use mo_carchm,        only: carchm
+
+    ! Arguments
+    integer, intent(in)    :: kpie                                            ! 1st dimension of model grid.
+    integer, intent(in)    :: kpje                                            ! 2nd dimension of model grid.
+    integer, intent(in)    :: kpke                                            ! 3rd (vertical) dimension of model grid.
+    integer, intent(in)    :: kbnd                                            ! number of halo grid points.
+    integer, intent(in)    :: kplyear                                         ! current year.
+    integer, intent(in)    :: kplmon                                          ! current month.
+    integer, intent(in)    :: kplday                                          ! current day.
+    integer, intent(in)    :: kldtday                                         ! number of time step in current day.
+    real,    intent(in)    :: pdlxp  (kpie,kpje)                              ! size of grid cell (longitudinal) [m].
+    real,    intent(in)    :: pdlyp  (kpie,kpje)                              ! size of grid cell (latitudinal) [m].
+    real,    intent(in)    :: pddpo  (kpie,kpje,kpke)                         ! size of grid cell (depth) [m].
+    real,    intent(in)    :: prho   (kpie,kpje,kpke)                         ! density [kg/m^3].
+    real,    intent(in)    :: pglat  (1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd)      ! latitude of grid cells [deg north].
+    real,    intent(in)    :: omask  (kpie,kpje)                              ! land/ocean mask.
+    real,    intent(in)    :: dust   (kpie,kpje)                              ! dust deposition flux [kg/m2/month].
+    real,    intent(in)    :: rivin  (kpie,kpje,nriv)                         ! riverine input [kmol m-2 yr-1].
+    real,    intent(in)    :: ndep   (kpie,kpje)                              ! nitrogen deposition [kmol m-2 yr-1].
+    real,    intent(in)    :: oafx   (kpie,kpje)
```

Comment:
`oafx` comment: `! alkalinity flux from alkalinization [kmol m-2 yr-1]` while `pi_ph` has no comment yet @JorgSchwinger , maybe you can add on this?

Reply:
I think that the comment `! alkalinity flux from alkalinization [kmol m-2 yr-1]`belongs with  the `oafx` variable. Right?

Reply:
Yes.